src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -21,9 +21,11 @@ internal class RequestTracingConstants
 
         public const string RequestTypeKey = "RequestType";
         public const string HostTypeKey = "Host";
-        public const string EnvironmentKey = "Environment";
-        public const string KeyVaultConfiguredTag = "KeyVaultConfigured";
-        public const string OfflineCacheConfiguredTag = "OfflineCacheConfigured";
+        public const string EnvironmentKey = "Env";
+        public const string DevEnvironmentValue = "Dev";
+        public const string KeyVaultConfiguredTag = "HasKeyVault";
```

Comment:
Maybe **Use**KeyVault and **Use**OfflineCache?

Reply:
In that case, we probably want `UsesKeyVault`?

Reply:
If we go with "Use", we should call it "**Using**KeyVault" or "**Uses**KeyVault". Both have more letters than "HasKeyVault". But its not a big difference. How about "UsesKeyVault" and "UsesOfflineCache" then?

Reply:
Ah, just saw Jimmy's comment. I can change it to "Uses..." then.